### PR TITLE
Own nickname to me

### PR DESCRIPTION
**Pull request by IndianaDschones**
**The problem:**
Instead of showing 'Me' in the message overview (if group message) the own nickname was shown. This was different than in the app.

**The solution**
A new displayName filter, which returns the displayName (Nickname) for non-me contacts and 'Me' for the own contact

Todo:
- [ ] Translations for 'Me' 

Idk how the translation process for new strings is handled - but you can tell me :)

### CODE REVIEW

**Comment by dbrgn** [comment]
Sorry for the late review, and thanks!

Looks good to me so far. The translations are handled by committing the new translation strings to the master branch, followed by community translations on [Transifex](https://www.transifex.com/threema-gmbh-1/threema-web/dashboard/).

Maybe you could also apply the filter in the group details page, where the group members are listed?

File: src/filters.ts [hunk 1]
```diff
@@ -431,4 +431,16 @@ angular.module('3ema.filters', [])
...
 
+/**
+ * Show 'Me' for own contact, for all other contacts show displayName
+ */
+.filter('displayName', ['WebClientService', '$translate',
+    function(webClientService: WebClientService, $translate: ng.translate.ITranslateService) {
+        return function(contact: threema.Receiver) {
+            if (contact.id === webClientService.me.id) {
+                return $translate.instant('messenger.ME');
+            } else { return contact.displayName; }
```
Comment (dbrgn) [inline src/filters.ts]: Could you put the return on its own line?

**Comment by IndianaDschones** [comment]
Yeah sure, i´ll look in to it this week :)

**Comment by dbrgn** [comment]
@IndianaDschones ping :slightly_smiling_face:

**Comment by IndianaDschones** [comment]
Pong 😃 
I also added the filter to the idsToNames filter. Now the conversation header including the group members contains "me" instad of the display name

**Comment by dbrgn** [comment]
Thanks!

**Pull request merged**